ideascube/serveradmin/catalog.py

```diff
@@ -604,9 +606,17 @@ def list_nothandled(self, ids):
 
         return sorted(pkgs, key=attrgetter('id'))
 
+    def update_displayed_packages_on_home(self, *, to_remove_ids=[], to_add_ids=[]):
```

Comment:
> Only if the arguments are changed in the function. Else it is useless.

It's also a big trap for the next person coming by.

Suppose we evolve this function and start modifying the argument? Then we'll need to remember to also change the default value... except we will forget, and the reviewer won't see it because the diff context won't show that line.

Using `[]` as a default kwargs value is just too risky.

As for `to_remove_ids = to_remove_ids or []` I don't like it any more than you, which is why I pretty much never use it. :smiley: 

I much prefer the explicit versions:

```
if to_remove_ids is None:
    to_remove_ids = []

to_remove_ids = to_remove_ids if to_remove_ids is not None else []
```